src/host/outputStream.cpp

```diff
@@ -454,3 +454,7 @@ void ConhostInternalGetSet::MarkCommandFinish(std::optional<unsigned int> /*erro
 {
     // Not implemented for conhost.
 }
+void ConhostInternalGetSet::InvokeMenu(std::wstring_view /*menuJson*/, unsigned int /*replaceLength*/)

```

Comment:
In general, and now in specific since this is conhost code, I don't think we should be using generic terms like "invoke menu" and "menu changed".

This is a very specific thing that, at these lower levels of the internal API, deserves a better name.

Somebody might misread this as "does something to invoke the console host system menu"

Reply:
Does `InvokeCompletions` feel better?